.env

```diff
@@ -0,0 +1,2 @@
+DOCSERVER_VERSION=latest
+SPEC_TYPE=
```

Comment:
I think this option should have default value, like `DOCSERVER_VERSION` to understand which option can be placed here

Reply:
The parameter is left blank because by default I want to run all the tests in the project with the `docker-compose up` command.

Reply:
In that case maybe add one example and comment it out